bundles/ch.elexis.core.data/src/ch/elexis/data/LabOrder.java

```diff
@@ -63,7 +63,7 @@ public static String getStateLabel(State state) {
 		case ORDERED:
 			return Messages.LabOrder_stateOrdered;
 		case DONE:
-			return Messages.LabOrder_stateDone;
+			return Messages.EditReminderDialog_fixed;
 		case DONE_IMPORT:
```

Comment:
Ich find das irgendwie nicht gut. Wenn beide `LabOrder_stateDone` sowie `EditReminderDialog_fixed` den gleichen Inhalt haben, dann sollte die Variable neutral heißen bzwp `Done`.  So ist das irgendwie komisch wenn im LabOrder eine Variable mit Bezeichnung EditReminderDialog verwendet wird. Ist aber nur eine Meinung, weiß nicht wie @huthomas das sieht.

Reply:
Das begreife ich. Um das zu ändern, müsste ich meine Script, um diese Anpassungen zu machen nochmals überarbeiten, um  in einer separaten Datei neu sprechende Namen zuzuweisen. Jetzt wir halt einfach  genommen was mit 'Core_' beginnt oder alphabetisch zuerst kommt.
Da ich für diese Arbeit schon Dutzende von Stunden investiert habe, die ich von Julian NICHT bezahlt bekomme, ist mein Enthusiasmus für eine tolle Lösung nicht sehr hoch.
Aber ich habe genügend Einzelfälle durchgecheckt, dass ich sehr zuversichtlich bin, dass das Script + meine persönlichen Änderungen korrekt sind.